examples/post.v3.LR.historical.zppy_v3.cfg

```diff
@@ -0,0 +1,289 @@
+# This is an example zppy v3.0.0 configuration file for Chrysalis.
+# v3.0.0 has several non-backward compatible ("breaking") changes.
+# These are described here.
+# In particular, there are a total of 5 deprecated parameters.
+# (Search for "DEPRECATED" in this file to find them.)
+# zppy v3.0.0 is included in E3SM Unified 1.11.0.
+
+[default]
+# Change these two paths to include your username instead
+# You can also change these to match your preferred directory structure
+output = /lcrc/group/e3sm/ac.forsyth2/E3SMv3_20250331_try2/v3.LR.historical_0051
+www = /lcrc/group/e3sm/public_html/diagnostic_output/ac.forsyth2/E3SMv3_20250331_try2
+# Once E3SM Unified 1.11.0 is released, you can use this line instead:
+# environment_commands = "source /lcrc/soft/climate/e3sm-unified/load_latest_e3sm_unified_chrysalis.sh"
+environment_commands = "source /lcrc/soft/climate/e3sm-unified/test_e3sm_unified_1.11.0rc13_chrysalis.sh"
```

Comment:
TODO: before merging, just make this the `environment_commands = "source /lcrc/soft/climate/e3sm-unified/load_latest_e3sm_unified_chrysalis.sh"` line.